docs/linting/TROUBLESHOOTING.md

```diff
@@ -86,6 +86,21 @@ module.exports = {
 };
 ```
 
+## I am running into errors when parsing TypeScript in my .vue files
+
+If you are running into issues parsing .vue files, it might be because parsers like [vue-eslint-parser](https://www.npmjs.com/package/vue-eslint-parser) are required to parse `.vue` files.  In this case you can move `@typescript-eslint/parser` inside `parserOptions` and use `vue-eslint-parser` as the top level parser.
+
+```diff
+- "parser": "@typescript-eslint/parser",
++ "parser": "vue-eslint-parser",
+  "parserOptions": {
++     "parser": "@typescript-eslint/parser",
+      "sourceType": "module"
+  }
+```
+
+The `parserOptions.parser` option can also specify an object to specify multiple parsers. See the [vue-eslint-parser README](https://github.com/vuejs/vue-eslint-parser#readme) for more details.
```

Comment:
The docs link you chose to link in the PR description is also really good: https://eslint.vuejs.org/user-guide/#usage. If that's the right place to send folks, maybe we should send them there instead?

```suggestion
The `parserOptions.parser` option can also specify an object to specify multiple parsers. See the [vue-eslint-parser usage guide](https://eslint.vuejs.org/user-guide/#usage) for more details.
```